REFERENCE: QGA-2 Create quiz model

sadinovsr opened the pull request:
DESCRIPTION: Created QuizModel

Commit:
REFERENCE: QGA-2
  DESCRIPTION: Created QuizModel

Gruseckis commented on models/QuizModel.js:
```diff
@@ -0,0 +1,26 @@
+
+const updateQuizById = async (id, model) => QuizModel.findByIdAndUpdate(id, model);
+
+const deleteQuizById = async _id => QuizModel.deleteOne({ _id });
```
Please change to findByIdAndDelete(id)

Gruseckis commented on models/QuizModel.js:
```diff
@@ -0,0 +1,26 @@
+
+const deleteQuizById = async _id => QuizModel.deleteOne({ _id });
+
+export { save, getAllQuizzes, getQuizById, getQuizzesByOwnerId, updateQuizById, deleteQuizById };
```
Please export quizSchema it will be used for unit tests

Commit:
REFERENCE: QGA-2
  DESCRIPTION: Changed deleteOne to findByIdAndDelete and added
    quizSchema to export.

Commit:
REFERENCE: QGA-2
  DESCRIPTION: Added description to quizSchema.
